src/gui/optionsdlg.ui

```diff
@@ -2782,6 +2782,22 @@ Use ';' to split multiple entries. Can use wildcard '*'.</string>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
+                  <widget class="QLabel" name="lblWebUiAddress">
+                   <property name="text">
+                    <string>IP Address:</string>
+                   </property>
+                  </widget>
+                 </item>
+                 <item>
+                  <widget class="QLineEdit" name="textWebUiAddress">
+                   <property name="toolTip">
+                    <string>Ip Address which the WebUi will bind to.
```

Comment:
* Ip Address -> IP address
* WebUi -> Web UI

Reply:
This doesn't seem right. Maybe
"The IP address that the Web UI will bind to."?